mock/src/test/java/com/tngtech/keycloakmock/impl/helper/TokenHelperTest.java

```diff
@@ -44,17 +48,17 @@
 
   @BeforeEach
   void setup() {
-    doReturn(CLIENT_ID).when(session).getClientId();
-    doReturn(SESSION_ID).when(session).getSessionId();
-    doReturn(NONCE).when(session).getNonce();
-    doReturn(USER).when(session).getUsername();
-    doReturn(ROLES).when(session).getRoles();
+    lenient().doReturn(CLIENT_ID).when(session).getClientId();
+    lenient().doReturn(SESSION_ID).when(session).getSessionId();
+    lenient().doReturn(NONCE).when(session).getNonce();
+    lenient().doReturn(USER).when(session).getUsername();
+    lenient().doReturn(ROLES).when(session).getRoles();
     doReturn(TOKEN).when(tokenGenerator).getToken(configCaptor.capture(), same(urlConfiguration));
```

Comment:
Please do not register captors during mock setup, but use them only during verification (see https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/Mockito.html#15).